src/Compilers/CSharp/Test/Symbol/Compilation/LoadDirectiveTests.cs

```diff
@@ -51,11 +51,11 @@ public void FileWithErrors()
             compilation.GetParseDiagnostics().Verify(
                 // a.csx(2,27): error CS1504: Source file 'b.csx' could not be opened -- Could not find file.
                 //                     #load "b.csx";
-                Diagnostic(ErrorCode.ERR_NoSourceFile, @"""b.csx""").WithArguments("b.csx", "Could not find file.").WithLocation(2, 27));
+                Diagnostic(ErrorCode.ERR_NoSourceFile, @"""b.csx""").WithArguments("b.csx", CSharpResources.CouldNotFindFile).WithLocation(2, 27));
```

Comment:
>CSharpResources.CouldNotFindFile [](start = 92, length = 32)

Is this string coming from exception? Please confirm that and, if that is the case, wrap the body of the test with ```using (new EnsureEnglishUICulture())```

Reply:
If the string is not coming from exception we need to take a closer look at what is going on here.

---
In reply to: [168914982](https://github.com/dotnet/roslyn/pull/24460#discussion_r168914982) [](ancestors = 168914982)